articles/native-platforms/ios-reactnative.md

```diff
@@ -199,8 +220,6 @@ RCT_EXPORT_METHOD(showTouchID:(NSDictionary *)options callback:(RCTResponseSende
 @end
 ```
 
-> You can also download [LockReactModule.h](https://raw.githubusercontent.com/auth0/native-mobile-samples/master/iOS/basic-sample-reactnative/iOS/Modules/LockReactModule.h) and [LockReactModule.m](https://raw.githubusercontent.com/auth0/native-mobile-samples/master/iOS/basic-sample-reactnative/iOS/Modules/LockReactModule.m) and add them to your Xcode project.
```

Comment:
I removed these links because mentioned files do not exist in the repository. Can you please provide me a place where I should upload or commit sample files to put correct links here?